sdk/monitor/monitor-query/swagger/metric-definitions.md

```diff
@@ -5,11 +5,11 @@
 ## Configuration
 
 ```yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/21f5332f2dc7437d1446edf240e9a3d4c90c6431/specification/monitor/resource-manager/Microsoft.Insights/stable/2018-01-01/metricDefinitions_API.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0550754fb421cd3a5859abf6713a542b682f626c/specification/monitor/resource-manager/Microsoft.Insights/stable/2023-10-01/metricDefinitions_API.json
```

Comment:
Same here - all metrics swaggers should use 2024-02-01 version

Reply:
@srnagar There is no `2024-02-01` folder at https://github.com/Azure/azure-rest-api-specs/tree/0550754fb421cd3a5859abf6713a542b682f626c/specification/monitor/resource-manager/Microsoft.Insights/stable. Can you please confirm?

Reply:
Done